Templating/Helper/LocaleHelper.php

```diff
@@ -43,7 +43,8 @@ public function language($code, $locale = null)
     {
         $codes = explode('_', $code);
 
-        $name = Intl::getLanguageBundle()->getLanguageName($codes[0], isset($codes[1]) ? $code[1] : null, $locale ?: $this->localeDetector->getLocale());
+        $name = Intl::getLanguageBundle()->getLanguageName($codes[0], isset($codes[1]) ? $codes[1] : null, 
+            $locale ?: $this->localeDetector->getLocale());
```

Comment:
Not sure this will pass cs validation. That's not how you're supposed to write a multiline call… here is what it should look like : 

```php
$name = Intl::getLanguageBundle()->getLanguageName(
    $codes[0],
    isset($codes[1]) ? $codes[1] : null,
    $locale ?: $this->localeDetector->getLocale()
);
```